### PR TITLE
Player forced movement component

### PR DESCRIPTION
**Pull request by aronwk-aaron**
Do not merge until after #671 is merged

Cleaned up whitespace on the components enum and removed commented out code

### CODE REVIEW

**Comment by EmosewaMC** [comment]
> Do not merge until after #671 is merged
> 
> Cleaned up whitespace on the components enum and removed commented out code

Should be marked as draft imo whole waiting for another pr

**Comment by aronwk-aaron** [comment]
> > Do not merge until after #671 is merged
> > 
> > Cleaned up whitespace on the components enum and removed commented out code
> 
> Should be marked as draft imo while waiting for another pr

¯\_(ツ)_/¯

File: dGame/dComponents/PlayerForcedMovementComponent.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include "Entity.h"
+#include "GameMessages.h"
```
Comment (EmosewaMC) [inline dGame/dComponents/PlayerForcedMovementComponent.h:4]: Is this include necessary?  doesn't look like its being used and having it in the header forces all files that inherit this one to inherit this header.

**Review by EmosewaMC — COMMENTED** [review]
Just one comment about a potentially pointless include that is in a header file

File: dGame/dComponents/PlayerForcedMovementComponent.h (hunk 1 above)
Comment (aronwk-aaron) [reply dGame/dComponents/PlayerForcedMovementComponent.h:4]: Will fix in future PR

**Pull request merged**